electron-react/src/modules/incoming_api.js

```diff
@@ -155,18 +155,19 @@ export const incomingReducer = (state = { ...initial_state }, action) => {
         }
       } else if (command === "get_wallet_balance") {
         if (data.success) {
-          id = data.wallet_id;
+          const wallet_balance = data.wallet_balance;
+          id = wallet_balance.wallet_id;
```

Comment:
I believe this is a typo.

```suggestion
          id = data.wallet_id;
```